src/Framework/MSBuildEventSource.cs

```diff
@@ -242,7 +240,7 @@ public void RarOverallStart()
             WriteEvent(27);
         }
 
-        [Event(28, Keywords = Keywords.All | Keywords.PerformanceLog)]
+        [Event(28, Keywords = Keywords.All | Keywords.PerformanceLog, Version = 1)]
```

Comment:
Why do we need this `Version = 1`

Reply:
The [rules for these events](https://learn.microsoft.com/dotnet/api/system.diagnostics.tracing.eventsource?view=net-7.0#conventions) say 

> Attempt to keep [EventSource](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.tracing.eventsource?view=net-7.0) objects backwards compatible and version them appropriately. The default version for an event is `0`. The version can be can be changed by setting [Version](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.tracing.eventattribute.version?view=net-7.0#system-diagnostics-tracing-eventattribute-version). Change the version of an event whenever you change properties of the payload. Always add new payload properties to the end of the event declaration. If this is not possible, create a new event with a new ID to replace the old one.

In #6410 I added fields to this event but didn't bump its version. Fixing that retroactively. (This was the only one I saw, looking through a `git blame` of this file.)